Sources/StreamVideo/CallKit/CallKitPushNotificationAdapter.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 import PushKit
 
 /// Handles push notifications for CallKit integration.
-open class CallKitPushNotificationAdapter: NSObject, PKPushRegistryDelegate, ObservableObject {
```

Comment:
https://github.com/GetStream/stream-video-swift/blob/776ed9dbb99bf9b72b6687c6550781b9411c720c/Sources/StreamVideo/CallKit/CallKitPushNotificationAdapter.swift#L172

Reply:
That was changed since your last comment, right? I can see that now it is
``swift
@preconcurrency open class CallKitPushNotificationAdapter: NSObject, ObservableObject, PKPushRegistryDelegate, @unchecked Sendable {
```